index.js

```diff
@@ -22,7 +22,25 @@ function getPercentageScore(score) {}
  * @param {object} score2 - An object representing another score of an assignment. Has the same shape as `score1`.
  * @returns {number} A rounded integer, representing the highest percentage scored among the two given scores.
  */
-function getHighestOfTwoScores(score1, score2) {}
+function getHighestOfTwoScores(score1, score2) {
+  const score = getPercentageScore
+  return score(score1) > score(score2) ? score(score1) : score(score2)
```

Comment:
Nice use of a ternary, but: you could make this even cleaner and more readable with one of the built-in Math functions. Figure out which one and you'll see your code tighten up!

Reply:
I always forget about the Math methods 😅 thanks